src/react-components/libraryDetailsView.tsx

```diff
@@ -43,15 +44,15 @@ export const LibraryDetailsView = ({ currentLibraryId }: LibraryDetailsViewProps
       {!books.loading && errorProp ? <div className="books-error-message"><ErrorView error={errorProp}/></div> : null}
       {isFormVisible ? (
         <section className="form-container">
-          <button className="hide-form-btn" onClick={hideForm}>Hide form</button>
           <FormView currentLibraryId={currentLibraryId} />
+          <button className="hide-form-btn" onClick={hideForm}>Cancel</button>
         </section>
       ) : <button className="addBookBtn" onClick={displayForm}>Add a Book</button>}
       {!books.loading && books.books.length && !errorProp ? (
```

Comment:
For future refactoring, we should consider removing the conditional logic from all of our component return statements. The `BooksDetailsView.tsx` file has one example of how to do this.